### DISCUSSION TITLE
Install openmptcp to a NG R7800

### DISCUSSION THREAD

**Discussion by vgnx** [Ysurac/openmptcprouter#3661] [Q&A]
Hi :wave: 

I'm trying to install openmptcprouter to a Netgear R7800.
Whether I use nmrpflash or the web UI of the router, it fails.
nmrpflash display the error "[TFTP block rollover. Upload might fail!](https://github.com/jclehner/nmrpflash?tab=readme-ov-file#tftp-block-rollover-upload-might-fail)" which means that the firmware size exceed the limit

But as the firware exists and some people [succeed to install it](https://github.com/Ysurac/openmptcprouter/discussions/2738), I wonder if there is a trick to install it.

Thank you for your help

**Comment by a834003831** [comment]
> 你好👋
> 
> 我正在尝试将openmptcprouter安装到Netgear R7800上。无论我用nmrpflash还是路由器的web UI，都失败。nmrpflash显示错误"[TFTP街区翻车。上传可能会失败！](https://github.com/jclehner/nmrpflash?tab=readme-ov-file#tftp-block-rollover-upload-might-fail)这意味着固件大小超出了限制
> 
> 但是由于防火墙的存在和一些人[成功安装](https://github.com/Ysurac/openmptcprouter/discussions/2738)，不知道有没有安装的窍门。
> 
> 谢谢你的帮助

Hello, I also have this device. May I ask if you have already compiled it into firmware? I would like to flash OMR onto this device as well, but I don’t know how to compile it.

**Reply by vgnx** [reply]
Hello, 
Sorry for the late answer.
I didn't compiled anything, I downloaded the firmware for this device in the [download page](https://www.openmptcprouter.com/download)

**Reply by vgnx** [reply]
@a834003831 FYI
https://github.com/Ysurac/openmptcprouter/issues/3726
